### PR TITLE
in_process_exporter_metrics: convert rss memory from pages to bytes

### PR DESCRIPTION
**Pull request by Bedledl**
https://github.com/fluent/fluent-bit/issues/9343
----
Enter `[N/A]` in the box, if an item is not applicable to your change.

**Testing**
Before we can approve your change; please submit the following in a comment:

- [ ] Example configuration file for the change
- [ ] Debug log output from testing the change

- [ ] Attached [Valgrind](https://valgrind.org/docs/manual/quick-start.html) output that shows no leaks or memory corruption was found

If this is a change to packaging of containers or native binaries then please confirm it works for all targets.

- [ ] Run [local packaging test](./packaging/local-build-all.sh) showing all targets (including any new ones) build.
- [ ] Set `ok-package-test` label to test for all targets (requires maintainer to do).

**Documentation**

- [ ] Documentation required for this feature

**Backporting**

- [ ] Backport to latest stable release.

----

Fluent Bit is licensed under Apache 2.0, by submitting this pull request I understand that this code will be released under the terms of that license.

### CODE REVIEW

**Comment by Bedledl** [comment]
Example configuration: `fluent-bit -i process_exporter_metrics -p metrics="memory" -o stdout`

Output thats shows successful fix: `process_memory_bytes{name="rsyslogd",pid="167",ppid="1",type="rss"} = 483328`

` cat /proc/167/status | grep Rss
RssAnon:             472 kB
RssFile:               0 kB
RssShmem:              0 kB`

**Review by cosmo0920 — COMMENTED** [review]
Basically, it works. I added a comment for the type of returning result of sysconf system call.

File: plugins/in_process_exporter_metrics/pe.h [hunk 1]
```diff
@@ -52,6 +52,7 @@ struct flb_pe {
     struct flb_input_instance *ins; /* input instance   */
     struct mk_list *metrics;        /* enabled metrics */
     int enabled_flag;               /* indicate enabled metrics */
+    int page_size;                  /* page size for converting number of pages to bytes */
```
Comment (cosmo0920) [inline plugins/in_process_exporter_metrics/pe.h:55]: sysconf(3) returns long type of value. So, do we need to use long instead of int here?

Comment (Bedledl) [reply plugins/in_process_exporter_metrics/pe.h]: I fixed it

**Comment by cosmo0920** [comment]
Could you use signed-off in your commit?
We use DCO and it complains signed-off is missing:

> Summary

> Commit sha: [09989e7](https://github.com/fluent/fluent-bit/pull/9350/commits/09989e73d2d16420cbb6133f48ce917913fddc58), Author: Bettina Riedl, Committer: Bettina Riedl; The sign-off is missing.

**Comment by Bedledl** [comment]
> Could you use signed-off in your commit? We use DCO and it complains signed-off is missing:
> 
> > Summary
> 
> > Commit sha: [09989e7](https://github.com/fluent/fluent-bit/pull/9350/commits/09989e73d2d16420cbb6133f48ce917913fddc58), Author: Bettina Riedl, Committer: Bettina Riedl; The sign-off is missing.

Added sign-off :)

**Review by cosmo0920 — APPROVED** [review]

**Comment by pawelrubin** [comment]
Hi all, any chance to get this fix merged?

**Review by leonardo-albertovich — APPROVED** [review]
Side notes : 

1. `sysconf` can fail and return `-1` but I don't think that's likely so I don't think requesting a change is necessary.
2. I'd rephrase line 1112 as `val *= ctx->page_size;` but that's just a matter of preference.
